hack/get-mcd-nodes.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+# This is a simple script that grabs all the cluster nodes and associates them
+# with a given MCD pod and outputs their roles. There is probably an easier way
+# to do this that I am too naïve to be aware of :).
```

Comment:
Yeah. It's more apparent if you run it. I assume this was done because `oc get node` dosn't have the pods and `oc get pods` has the nodes but not the roles.
```
machine-config-daemon-9rf62	jkyros-fixed-buildcon-nqbcv-master-1      	master
machine-config-daemon-ggw9l	jkyros-fixed-buildcon-nqbcv-master-0      	master
machine-config-daemon-jjsql	jkyros-fixed-buildcon-nqbcv-master-2      	master
machine-config-daemon-nspjb	jkyros-fixed-buildcon-nqbcv-worker-c-z8zjg	worker
machine-config-daemon-sllxp	jkyros-fixed-buildcon-nqbcv-worker-d-hv4qb	worker
machine-config-daemon-wzvgk	jkyros-fixed-buildcon-nqbcv-worker-b-bkvtr	worker
```

Reply:
Yeah, I'll update that comment and include this sample output in said comment.